libs/platform/core/src/services/page-meta/page-meta.service.ts

```diff
@@ -0,0 +1,15 @@
+import { ElementAttributes, ElementDefinition } from './page-meta.model';
+
+export const PageMetaService = 'oryx.PageMetaService';
+
+export interface PageMetaService {
+  add(definitions: ElementDefinition | ElementDefinition[]): void;
+  update(definition: ElementDefinition): void;
+  setAttributes(attrs: ElementAttributes, element: HTMLElement): void;
```

Comment:
To avoid confusion and overusing this one, I'd chage it either to
```suggestion
  setHtmlAttributes(attrs: ElementAttributes): void;
```
or limiting elements
```suggestion
  setAttributes(attrs: ElementAttributes, element: 'html' | 'body'): void;
```